src/js/tech/tech.js

```diff
@@ -33,10 +34,20 @@ import document from 'global/document';
 class Tech extends Component {
 
   constructor(options={}, ready=function(){}){
+    let defaults = {
```

Comment:
by default all of the features are on if they are supported. If the user wants to then disable it, even if the feature was natively supported they could. I guess we don't really need defaults if we just want to check wether the option to keep nativeX on was passed in.